### PR TITLE
Thomas' Turtlehack

### PR DESCRIPTION
**Pull request by grassycheetah94**

### CODE REVIEW

File: _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+jack.fill(False)
+jack.penup()
+jack.goto (0,-150)
+jack.penup()
+```
+
+The user is asked their name and the ‘man’ responds to this answer by greeting him, then asking the weather. Then responding to that response, he answers in the affirmative. Then the ‘speaking man’ says he has to go and BYE! As seen below.
+
+```
+#Whats your name?
```
Comment (lisetted) [inline _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md:135]: Look! I can put a comment in your code!

**Comment by grassycheetah94** [comment]
@lisetted helped me figure out my problem and she suggested using '+' and it worked.

File: _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+author: grassycheetah94
+title: "Thomas' Turtlehack"
+---
+
+Here is the program I am embedding:
+<iframe src="https://trinket.io/embed/python/2a682241b6" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+---
+Reflection:
+I wanted to create a turtle version of the ‘Hello Chuck’ assignment we had this weekend. I started with creating a ‘man’. This took a lot longer than I originally thought it would, so he is not as detailed as I wanted him to be.  But the basic concept is still there. As you can see below.
```
Comment (eah13) [inline _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md:12]: Great idea to pick a previous assignment as a starting point and add a (very detailed!) drawing to it.

File: _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+jack.goto(-37,65)
+jack.goto(-80,65)
+jack.goto(-80,60)
+jack.goto(37,60)
+jack.fill(False)
+jack.penup()
+jack.goto (0,-150)
+jack.penup()
+```
+
```
Comment (eah13) [inline _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md:131]: Phew!  I do like it when students include code in a reflection, but you might consider pulling out just the pieces that are most relevant.  These long code blocks can be difficult to read.

File: _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+ell.goto(40,-105)
+ell.write("I have to go.", None, "center", "15pt bold")
+ell.goto(40,-130)
+ell.write("BYE!", None, "center", "15pt bold")
+```
+
+That looks like:
+![Turtle image](http://i.imgur.com/raFvMnq.png)
+![Turtle image](http://i.imgur.com/xs8iIOu.png)
+![Turtle image](http://i.imgur.com/lSntLnU.png)
```
Comment (eah13) [inline _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md:202]: Nice use of images!

For accessibility, please title the text inside the `[]` as a description of the image.  That's what screen readers will use to describe the image.

File: _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+The only problem I ran into was figuring out how to have user input be placed into a sentence along with non-user inputted words. So I was forced to figure out correct spacing between user imputted words and auto imputted words. Which, actually I like better.
```
Comment (eah13) [inline _posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md:204]: I like that you've identified a problem you encountered, but I'd like to have seen a little more unpacking here.  What did you do to figure out the correct spacing?  What was the code that did it (for one example perhaps)?  This is the most interesting part of your reflection, so focus on it!

**Comment by eah13** [comment]
Overall, very nicely done!  Your drawing was very precise, and I think you stopped at a reasonable point.  Even though you had grander plans for your talking man, the animation you have is nicely done and well commented.

Th input to let the user interact and have the man react to the user's input visually is creative.  

I do wish you'd zoomed in on the challenge a little and how you fixed it.  It sounds like from the comment that you made that Lisette helped you with string concatenation, so maybe this problem was resolved too late in the process to comment on, but either way, this is the kind of thing to describe fully in the future.

👍

**Pull request merged**